brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced columns - Border, overlay conflict and stack on issue.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/2121e94d-d987-4ff5-bc76-6c220614e1bb?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create an advanced column with 3 child in master
- add shape divider, border, box shadow, and stack on mobile.
- Switch to next-release and check front end the issue mention on infinity will be reproduced.
- now switch to above fix branch and refresh the frontend issue will resolved.

### Checklist:
